packages/react-accordion/src/components/Accordion/useAccordion.ts

```diff
@@ -76,12 +70,14 @@ function initializeUncontrolledOpenItems({
  * Updates the list of open indexes based on an index that changes
  * @param value - the index that will change
  * @param previousOpenItems - list of current open indexes
- * @param param2 - {multiple, collapsible}
+ * @param multiple - if Accordion support multiple Panels opened at the same time
+ * @param collapsible - if Accordion support multiple Panels closed at the same time
  */
 function updateOpenItems(
   value: AccordionItemValue,
   previousOpenItems: AccordionItemValue[],
-  { multiple, collapsible }: Pick<AccordionState, 'multiple' | 'collapsible'>,
+  multiple: boolean,
+  collapsible: boolean,
```

Comment:
Why was this change necessary ?

Reply:
I can revert it if you'd like! I was wondering if `multiple` and `collapsible` were only passed in as an object so they could be typed as `AccordionCommons`. I could also do a `Required<Pick<AccordionProps, 'multiple' | 'collapsible'>>`, but that seemed a little over-engineered for two boolean props. I could also type them as `{ multiple: boolean; collapsible: boolean; }` if there was another reason to pass them in as an object.

@ling1726 do you have a preference on the approach?

Reply:
ok I see, you wanted to avoid using the TS helpers here. I'm ok with that